L1Trigger/L1THGCal/src/HGCalTriggerCellCalibration.cc

```diff
@@ -31,8 +38,17 @@ void HGCalTriggerCellCalibration::calibrateInMipT(l1t::HGCalTriggerCell& trgCell
   // fCperMIP and thickCorr here
   /* convert the charge amplitude in MIP: */
   double trgCellMipP = amplitude;
-  if (isSilicon && fCperMIP_ > 0) {
-    trgCellMipP /= fCperMIP_;
+
+  if (DetId(trgCell.detId()).det() == DetId::HGCalTrigger and
+      HGCalTriggerDetId(trgCell.detId()).subdet() == HGCalTriggerSubdetector::HFNoseTrigger) {
+    if (fCperMIPnose_ > 0) {
+      trgCellMipP /= fCperMIPnose_;
+    }
+
+  } else {
+    if (isSilicon && fCperMIP_ > 0) {
```

Comment:
Can be `else if` in one single statement

Reply:
done